common/templates/common/selection_drugs.html

```diff
@@ -187,6 +187,18 @@
         width: 100%;
         text-align: center;
     }
+
+    /* Ensure the struct shows up properly */
+    #struct {
```

Comment:
Shouldn't #struct be a css class instead of a css id? If I am not mistaken multiple ligands are shown on this page which would mean multiple ligand structures.

Reply:
Also, I see it defined further down in ligand_complex_models.html. Please put this struct definition in a css file in the static folder (new or existing) and import it here and also in the other file.